### ISSUE TITLE
Model short Leios over large networks

### ISSUE THREAD

**Issue by SupernaviX** [input-output-hk/ouroboros-leios#49]
This effort is to write a model of "Short Leios" which runs against a realistic (several-thousand node) model of the real network. Short Leios is a simpler version of the full Leios protocol (without some historical state), which has lower throughput than Full Leios but is easier to understand. That makes it an ideal first target to model.

The goals are
- use visualizations of the sim's output as a learning tool, to refine human intuition around the protocol
- understand the resource requirements to run a node in this network
- experiment with the effect of changing parameters

**Comment by SupernaviX** [comment]
We have a model of Short Leios which we've run against a realistic number of SPOs, though we need to simulate an order of magnitude more relay nodes for true accuracy. 

We have basic visualizations (which have helped diagnose bugs in the simulation as well as tune parameter values), though the visualizations need some work before they can be truly educational. 

The simulation has enough detail to understand the effect of changing Leios parameters (or system resources) on transaction throughput. It is already useful for e.g. finding an optimal vote threshold for networks of a given size, but could still use more detail (in particular, the network model is still very basic).

Basically, I think that we've technically met all of these goals, but we'd like to keep addressing them in the next PI ( #150 )

**Comment by will-break-it** [comment]
Work completed
